packages/openapi/api.yaml

```diff
@@ -1,95 +1,61 @@
 components:
```

Comment:
you will want to revert this

Reply:
I just forced pushed without this file, let's see if the check fails, maybe it failed last time b/c of the conflict in the file